sale_commission/models/sale_commission.py

```diff
@@ -36,6 +42,70 @@ class SaleCommission(models.Model):
     sections = fields.One2many(
         comodel_name="sale.commission.section", inverse_name="commission")
     active = fields.Boolean(default=True)
+    period = fields.Selection(
+        selection=[(PERIOD_MONTH, "Monthly"),
+                   (PERIOD_QUARTER, "Every quarter"),
+                   (PERIOD_SEMI, "Every semester"),
+                   (PERIOD_YEAR, "Every year")],
+        required=True, default=PERIOD_MONTH,
+    )
+    scope = fields.Selection(
```

Comment:
For both the user clarity and code readability, this field should be split in 2: one for the "real" scope: company or own sale, and another for where to apply it: gross sale or margin. Margin should have indicated that it's list price minus cost price.